src/base/utils/version.h

```diff
@@ -50,7 +50,7 @@ namespace Utils
         typedef Version<T, N, Mandatory> ThisType;
 
         constexpr Version()
-            : m_components {}
+            : m_components{{}}
```

Comment:
@Piccirello, don't forget about Coding Style! In both cases. Besides, I've already mentioned it in my comments. Space should be before brace: `m_components {{}}` and `ComponentsArray res {{}};`.

Reply:
Whoops! Just updated the commit.